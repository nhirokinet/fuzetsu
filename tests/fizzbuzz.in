5
1
3
413
55
1500
