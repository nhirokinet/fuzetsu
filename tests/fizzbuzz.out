1
Fizz
413
Buzz
FizzBuzz
